yarn.lock

```diff
@@ -430,11 +430,56 @@
   resolved "https://registry.yarnpkg.com/@eslint/js/-/js-8.48.0.tgz#642633964e217905436033a2bd08bf322849b7fb"
   integrity sha512-ZSjtmelB7IJfWD2Fvb7+Z+ChTIKWq6kjda95fLcQKNS5aheVHn4IkfgRQE3sIIzTcSLwLcLZUD9UBt+V7+h+Pw==
 
+"@floating-ui/core@^1.0.0", "@floating-ui/core@^1.5.3":
```

Comment:
can you revert changes on this file too? i'll apply them myself as its easier than reviewing this